__tests__/fake-app.ts

```diff
@@ -140,3 +142,13 @@ export function onDisconnect(body) {
   // User unlinked their account, stop reporting state for the user
   return {};
 }
+
+describe.only('Disconnect response testing suite', () => {
```

Comment:
This confuses me. Why would developers put the test into their app code instead of their unit tests?

Reply:
Right now the fakeapp runs as a testing suite and fails since there are no tests. To avoid this, I've been doing a skip test here in my own code, but Nick suggested it might be a good idea to have an actual test in there instead.

Reply:
OK, that's probably an issue with the test framework configuration assuming that every `.ts` file should have tests in it. So the appropriate correction is probably to fix that.

Reply:
Something I've tried doing is looking into the jest-config files, but seems to be a problem that other people have been having trouble with too. Suggestions to ignore files by adding some certain lines of code in the config file hasn't been working for me. 

One suggestion I saw was to do npm test -- --testPathIgnorePatterns <ts file to be ignored here>, but not sure if that's the best way to go.  Thoughts?

Reply:
You just need to configure Jest to find the right files. Looks like the `testMatch` attribute does exactly this, so something like the following in your `jest.config.js` file would only run tests in files with the `test.ts` suffix:

`testMatch: ['**/__tests__/**/*.test.ts']`

Reply:
Oh perfect! Didn't catch this option. Thank you!